modules/storage-persistent-storage-pv.adoc

```diff
@@ -155,14 +155,14 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 |Red Hat OpenShift Container Storage  | ✅ | - | ✅
 |VMware vSphere | ✅ | - |  -
 endif::[]
-|===
 
-[NOTE]
-====
-Use a recreate deployment strategy for Pods that rely on AWS EBS.
+|===
+[.small]
+--
+1. ReadWriteOnce (RWO) volumes cannot be mounted on multiple nodes. If a node fails, the system does not allow the attached RWO volume to be mounted on a new node because it is already assigned to the failed node. If you encounter a multi-attach error message as a result, you can either recover or delete the node to make the volume available to other nodes. 
```

Comment:
s/encounter/receive ??

Do users know what _receover_ means? I don't see it mentioned in the topic. (Likely my own ignorance here)

Reply:
_Receive_ might be too passive here. _Recover_ seems to be the right term because the user needs to either recover the node that failed, or delete that node, to make the volume available. But you're right, it's not mentioned elsewhere in the topic, as this is a situation specific only to RWO volumes. 

With [OSDOCS-1299](https://issues.redhat.com/browse/OSDOCS-1299), I should be able to add additional instructions under Troubleshooting that can provide more details about recovering or deleting the failed node.